src/ASM/LR/ASMu3D.C

```diff
@@ -1542,12 +1542,14 @@ bool ASMu3D::evalSolution (Matrix& sField, const Vector& locSol,
     // Fetch element containing evaluation point.
     // Sadly, points are not always ordered in the same way as the elements.
     int iel = lrspline->getElementContaining(gpar[0][i],gpar[1][i],gpar[2][i]);
+    const LR::Element* el = lrspline->getElement(iel);
```

Comment:
Should perhaps move this to _after_ the `if (iel < 0)` check, just in case `lrspline->getElement()` does not handle a negative argument gracefully

Reply:
yup